Tests/GitHubTeams.tests.ps1

```diff
@@ -11,13 +11,516 @@
     Justification='Suppress false positives in Pester code blocks')]
 param()
 
+Set-StrictMode -Version 1.0
+
 # This is common test code setup logic for all Pester test files
 $moduleRootPath = Split-Path -Path $PSScriptRoot -Parent
 . (Join-Path -Path $moduleRootPath -ChildPath 'Tests\Common.ps1')
 
 try
 {
-    # TODO once more capabilities exist in the module's API set
+    # Define Script-scoped, readonly, hidden variables.
+    @{
+        defaultRepoDesc = "This is a description."
+        defaultRepoHomePage = "https://www.microsoft.com/"
+        defaultRepoTopic = "microsoft"
+    }.GetEnumerator() | ForEach-Object {
+        Set-Variable -Force -Scope Script -Option ReadOnly -Visibility Private -Name $_.Key -Value $_.Value
+    }
+
+    Describe 'GitHubTeams\Get-GitHubTeam' {
+
+        Context 'When getting a GitHub Team by organization' {
+            BeforeAll {
+                $repoName = [Guid]::NewGuid().Guid
+
+                $newGithubRepositoryParms = @{
+                    RepositoryName = $repoName
+                    OrganizationName = $script:organizationName
+                }
+                $repo = New-GitHubRepository @newGitHubRepositoryParms
```

Comment:
nit: line feed, here and below, separating the params from the execution line.  I generally prefer a linefeed after any closing expression, but with some exceptions have been allowing that to be avoided in the case of repeated expressions that are configuring a params hashtable.